src/intervals/construction.jl

```diff
@@ -64,18 +64,11 @@ struct Interval{T} <: Real
     lo::T
     hi::T
 
-    function Interval{T}(a, b) where T
```

Comment:
I dont thinks this should be removed? I think this is needed to correctly construct an interval from rationals or irrational numbers?

Reply:
Yes, in fact I did not delete it, I simply defined it as an outer constructor (cf. line 71) since it is more of a "feature" and not an "essential" constructor.

Reply:
I think that's fair. And since the tests don't complain it looks fine to me.

Besides, it clarifies the logic which is nice.